iceoryx_posh/testing/include/iceoryx_posh/testing/mocks/posh_runtime_mock.hpp

```diff
@@ -28,10 +28,10 @@ class PoshRuntimeMock : public iox::runtime::PoshRuntime
     static std::unique_ptr<PoshRuntimeMock> create(const iox::RuntimeName_t& name)
     {
         auto& runtime = mockRuntime();
-        iox::cxx::Expects(!runtime.has_value() && "Using multiple PoshRuntimeMock in parallel is not supported!");
-        iox::cxx::Expects(PoshRuntime::getRuntimeFactory() == PoshRuntime::defaultRuntimeFactory
-                          && "The PoshRuntimeMock can only be used in combination with the "
-                             "PoshRuntime::defaultRuntimeFactory! Someone else already switched the factory!");
+        IOX_EXPECTS(!runtime.has_value() && "Using multiple PoshRuntimeMock in parallel is not supported!");
+        IOX_EXPECTS(PoshRuntime::getRuntimeFactory() == PoshRuntime::defaultRuntimeFactory
```

Comment:
Why not use `WITH_MSG` here?

Reply:
I just did a mass search and replace. This will be fixed when we move to the macros defined in `error_reporting_macros.hpp`